### PR TITLE
Add an option to disable TitledPane animation

### LINKED ISSUE #171: Disable accordion menu animation
Originally reported by: **Jesper Öqvist (Bitbucket: [joqvist](https://bitbucket.org/joqvist), GitHub: [joqvist](https://github.com/joqvist))**

----------------------------------------

I would like a way to disable the accordion panel animation in SceneBuilder (used for Properties, Layout, etc.).

I think the animations are slow and annoying. I would like to switch between Properties and Layout instantly. An option in the SceneBuilder preferences for switching off the animations would be great!

----------------------------------------
- Bitbucket: https://bitbucket.org/gluon-oss/scenebuilder/issue/171

### PR DESCRIPTION
**Pull request by abhinayagarwal**
Adds an option in "Preferences" to disable animation in TitledPane(s) of Accordions through Scene Builder.

This PR will fix #171

### CODE REVIEW

File: app/src/main/resources/com/oracle/javafx/scenebuilder/app/i18n/SceneBuilderApp.properties [hunk 1]
```diff
@@ -239,6 +239,7 @@ prefs.cssanalyzer.columns.order = CSS Analyzer Columns order :
 prefs.cssanalyzer.columns.defaults.first = "Defaults" Column First
 prefs.cssanalyzer.columns.defaults.last = "Defaults" Column Last
 prefs.recent.items = Recent items
+prefs.animate.accordion = Animate Accordion
```
Comment (jperedadnr) [inline app/src/main/resources/com/oracle/javafx/scenebuilder/app/i18n/SceneBuilderApp.properties]: I'd say: `Enable accordion animations`?

File: app/src/main/resources/com/oracle/javafx/scenebuilder/app/preferences/Preferences.fxml [hunk 2]
```diff
@@ -110,9 +110,12 @@
       </items>
     </ChoiceBox>
 
-    <Separator prefWidth="-1.0" GridPane.columnIndex="0" GridPane.columnSpan="2" GridPane.rowIndex="16" />
+    <Label text="%prefs.animate.accordion" GridPane.columnIndex="0" GridPane.rowIndex="16" />
```
Comment (jperedadnr) [inline app/src/main/resources/com/oracle/javafx/scenebuilder/app/preferences/Preferences.fxml]: Maybe a new separator is required in between the number of recent items and the new accordion option, as these two are not related.

File: app/src/main/java/com/oracle/javafx/scenebuilder/app/preferences/PreferencesRecordGlobal.java [hunk 3]
```diff
@@ -502,6 +519,9 @@ public void writeToJavaPreferences(String key) {
                     applicationRootPreferences.put(IMPORTED_GLUON_JARS, stringBuilder.toString());
                 }
                 break;
+            case ACCORDION_ANIMATION:
+                applicationRootPreferences.put(ACCORDION_ANIMATION, String.valueOf(isAccordionAnimation()));
```
Comment (tiainen) [inline app/src/main/java/com/oracle/javafx/scenebuilder/app/preferences/PreferencesRecordGlobal.java]: You can use `Preferences.putBoolean()` to store boolean values.

File: app/src/main/resources/com/oracle/javafx/scenebuilder/app/i18n/SceneBuilderApp.properties (hunk 1 above)
Comment (abhinayagarwal) [reply app/src/main/resources/com/oracle/javafx/scenebuilder/app/i18n/SceneBuilderApp.properties]: It's a checkbox. Isn't "enable" implied? :)
